test/scripts/streaming/test-streaming-detection.py

```
@@ -0,0 +1,679 @@
+Olla Streaming Detection Test Script
+Detects whether Olla is streaming or buffering responses by analyzing chunk timing patterns
+"""
+
+import sys
+import json
+import time
+import argparse
+import requests
+import os
+from typing import Dict, List, Tuple, Optional, Any
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)